meson.build

```diff
@@ -31,8 +32,16 @@ meson_version = meson.version()
 
 cc = meson.get_compiler('c')
 os_name = host_machine.system()
+cpu_family = host_machine.cpu_family()
 cc_name = cc.get_id()
 
+python = find_program('python3')
+if meson_version.version_compare('>=0.55')
+  python_path = python.full_path()
+else
+  python_path = python.path()
+endif
```

Comment:
I neglected to re-review this before, but... why do you need this?

Reply:
For `vcs_tag`, and also knowing that I'll be needed for tests.